File: Hayaa.ServiceFrame/ServiceFrameClient/Hayaa.ServicePlatform.Client/Model/AppAuthReponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.ServicePlatform.Client.Model
{
   internal class AppAuthReponse
    {
        /// <summary>
        /// 申请到的实例Id
        /// </summary>
        public int AppInstanceId { set; get; }
        /// <summary>
        /// 申请的实例token
        /// </summary>
        public String AppInstanceToken { set; get; }
    }
}

File: Hayaa.ServiceFrame/ServiceFrameClient/hayaa.serviceplatform.client_java/hayaa.serviceplatform.client/src/main/java/hayaa/serviceplatform/client/AppAuthReponse.java
package hayaa.serviceplatform.client;

public class AppAuthReponse {
    private int AppInstanceId;
    private String AppInstanceToken;

    public int getAppInstanceId() {
        return AppInstanceId;
    }

    public void setAppInstanceId(int appInstanceId) {
        AppInstanceId = appInstanceId;
    }

    public String getAppInstanceToken() {
        return AppInstanceToken;
    }

    public void setAppInstanceToken(String appInstanceToken) {
        AppInstanceToken = appInstanceToken;
    }


}
